lib/usdUfe/ufe/UsdUndoPayloadCommand.cpp

```diff
@@ -70,22 +56,39 @@ void UsdUndoLoadUnloadBaseCommand::saveModifiedLoadRules() const
     UsdUfe::saveStageLoadRules(_stage);
 }
 
+// Macros to handle the redo/undo actions on stage load/unload rules
+// The load rules are saved before applying new rules and are
+// applied back when the commands are undo
+#define STAGE_PAYLOAD_ACTION(action)     \
```

Comment:
Could use a function instead of a macro - with a lambda you can do the same thing. Personnally I find macros annoying to maintain and work with.